chia/protocols/shared_protocol.py

```diff
@@ -17,6 +17,7 @@
 # These are passed in as uint16 into the Handshake
 class Capability(IntEnum):
     BASE = 1  # Base capability just means it supports the chia protocol at mainnet
+    BLOCK_HEADERS = 2
```

Comment:
we have some name clashing here, we had `RequestHeaderBlocks` (old api call) and `RespondBlockHeader`, and new api call is `RequestBlockHeaders` 

naming is hard 🤷